test/FunctionalTestUtils/InProcessServer.cs

```diff
@@ -53,14 +52,15 @@ private BackTelemetryChannel Start(string assemblyName)
             configBuilder.Add(customConfig);
             var config = configBuilder.Build();
 
-            var engine = CreateBuilder(config)
-                .UseServer("Microsoft.AspNet.Server.WebListener")
+            this.hostingEngine = CreateBuilder(config)
```

Comment:
There's a much cleaner way to set the URL in the new hosting API. You can get rid of the config goop.